qubesmanager/settings.py

```diff
@@ -1498,6 +1498,12 @@ def delete_rule_button_pressed(self):
 def main(args=None):
     args = parser.parse_args(args)
     vm = args.domains.pop()
+    if vm.klass == 'AdminVM':
+        print("This tool cannot be used to change properties of an "
+              f"AdminVM ({vm.name}).")
+        print("You can use command-line tools such as qvm-prefs "
+              "and qvm-features to change properties of an AdminVM")
+        return 1
```

Comment:
For this to take effect, `main()` call below needs to use the return value, like `sys.exit(main())`